lib/src/core/guild/guild.dart

```diff
@@ -520,6 +520,9 @@ class Guild extends SnowflakeEntity implements IGuild {
       raw["emojis"].forEach((dynamic o) {
         final emoji = GuildEmoji(client, o as RawApiMap, id);
         emojis[emoji.id] = emoji;
+        if (client.cacheOptions.emojiCachePolicy.canCache(emoji) && client.cacheOptions.emojiCachePolicyLocation.http) {
+          client.emojis[emoji.id] = emoji;
+        }
```

Comment:
why we need global cache for emojis?

Reply:
Even if we need that kind of cache it would be better to interate over guilds are flatmap all emojis from guilds instead creating another cache storage

Reply:
Also emoji cache policies and thingies are not applied to caching emojis inside guild object which is non consistent with rest of your implementation and whole lib

Reply:
> why we need global cache for emojis?

I thought it would be useful to access directly by the client

Reply:
Yeah right, so I'll remove this property